app/src/main/java/com/example/oddhours/data/repository/JobRepository.kt

```diff
@@ -80,9 +78,8 @@ class JobRepository() {
     }
 
     fun getShiftsForUIList(): MutableList<ShiftsListModel> {
-        val jobIDList = getAllJobIDs()
+        val jobIDList = getAllJobs()
```

Comment:
maybe rename `jobIDList` as it's still a bit misleading as to what it contains. something like `jobsList`?

Reply:
oh right, i'll update that